docs/xml/catalog-iconcache.xml

```diff
@@ -20,9 +20,10 @@
 	<section id="spec-iconcache-location">
 		<title>Filesystem locations</title>
 		<para>
-			All icons of type <literal>cached</literal> must be placed in <filename>/usr/share/swcatalog/icons/%{origin}/%{size}/</filename> or <filename>/var/cache/swcatalog/icons/%{origin}/%{size}/</filename>,
+			All icons of type <literal>cached</literal> must be placed in <filename>/usr/share/swcatalog/icons/%{origin}/%{size}/</filename> or <filename>/var/lib/swcatalog/icons/%{origin}/%{size}/</filename>,
```

Comment:
`/var/cache/` is still a valid location, so maybe change this to 
```suggestion
			All icons of type <literal>cached</literal> must be placed in <filename>/usr/share/swcatalog/icons/%{origin}/%{size}/</filename> or <filename>/var/(lib|cache)/swcatalog/icons/%{origin}/%{size}/</filename>,
```

Reply:
I would be hesitant to add this change, since the `/var/cache` location is not mentioned anywhere else in the document, including nowhere in the catalog xml or yaml documentation. I would either keep `/var/cache` out of the equation as it currently is, or properly document this currently "edge-case" location. But you are the maintainer, so if you insist, I'll just add the suggestion :)

Reply:
You know, `/var/cache` is actually de-facto supported, but maybe there is a chance to phase out support for it... Yet, on the line you changed you changed "cache" to "lib", so I think we need to keep both locations mentioned here for now, until we actually commit to removing one.

Reply:
Fair enough, I'm always too fast at removing things. Updates as suggested :)